telegram/ext/filters.py

```diff
@@ -2147,14 +2148,24 @@ def filter(self, message: Message) -> bool:
     # neither mask nor emoji can be a message.sticker, so no filters for them
 
 
-class _SuccessfulPayment(MessageFilter):
-    __slots__ = ()
+class SuccessfulPayment(MessageFilter):
+    __slots__ = ("strings",)
+
+    def __init__(self, strings: Optional[Union[List[str], Tuple[str, ...]]] = None):
```

Comment:
```suggestion
    def __init__(self, invoice_payloads: Optional[Union[List[str], Tuple[str, ...]]] = None):
```
explicit naming is better. Change it below also ofc